src/pages/register-a-beacon/check-your-answers.tsx

```diff
@@ -27,12 +27,14 @@ const CheckYourAnswersPage: FunctionComponent<CheckYourAnswersProps> = ({
 }: CheckYourAnswersProps): JSX.Element => {
   const pageHeading = "Check your answers before sending in your registration";
 
+  // TODO: This page is removed in PR check your answers.
```

Comment:
Sorry if I'm missing something, but is this comment meant to be on the `check-beacon-summary` page?

I'm guessing we're talking about [this PR](https://github.com/mcagov/beacons-webapp/pull/137/files)?

Reply:
Good spot @Ninamma Absolutely that yes. 

This file is updated in that PR and `check-beacon-summary` is removed in PR